src/assets/src/components/addQueue.tsx

```diff
@@ -179,7 +186,14 @@ export function AddQueuePage(props: PageProps) {
             ? validateAndSetDescriptResult(description) : descriptValidationResult;
         const curAllowedValidationResult = !allowedValidationResult
             ? validateAndSetAllowedResult(allowedMeetingTypes) : allowedValidationResult;
-        if (!curNameValidationResult!.isInvalid && !curDescriptValidationResult!.isInvalid && !curAllowedValidationResult!.isInvalid) {
+        const curLocationValidationResult = !locationValidationResult 
+            ? validateAndSetLocationResult(physLocation) : locationValidationResult;
+        if (
+            !curNameValidationResult!.isInvalid && 
+            !curDescriptValidationResult!.isInvalid && 
+            !curAllowedValidationResult!.isInvalid && 
+            !curLocationValidationResult.isInvalid
```

Comment:
Can you check whether the `!` assertion before the `.isInvalid` is required? It might not be. You're not using it in the one you added.